test/access-control.integration.js

```diff
@@ -324,4 +332,23 @@ describe('access control - integration', function() {
       return '/api/accounts-updating/' + this.accountWithReplaceOnPUTfalse.id + '/replace';
     }
   });
+
+  describe('/accounts with upsertWithWhere', function() {
+    var count = 0;
+    before(function() {
+      var roleModel = loopback.getModelByType(loopback.Role);
+      roleModel.registerResolver('$dummy', function(role, context, callback) {
+        process.nextTick(function() {
+          if (context.remotingContext) {
+            count++;
+          }
+          callback && callback(null, false); // Always true
+        });
+      });
+    });
+    lt.beforeEach.givenModel('accountWithReplaceOnPUTtrue');
```

Comment:
>there is already account.json and inside which the "name" is specified as "accountWithReplaceOnPUTtrue" and plural as "accounts-replacing". 

Oh right; by default LoopBack 3.0 sets `replaceOnPUT` to true, whereas in LoopBack 2.x it is set to false by default; so that's why `account.json` has a name property `accountWithReplaceOnPUTtrue`. How about using other available models in `test/fixtures/access-control/common/models`; probably you can use `bank` model for instance. I do not have a strong opinion I leave it to you and @bajtos.